### DISCUSSION TITLE
How to reopen GitHub Security Alert?

### DISCUSSION THREAD

**Discussion by dekisr** [community/community#22413] [New to GitHub]
<p>Hello there,</p>
<p>Recently I dismissed a GitHub vulnerabilty alert.</p>
<p>Is there any way to reopen it?</p>
<p>thanks in advance</p>

**Comment by mpboom** [comment, marked as answer]
<p>Sure, you can always access them trough the  <strong>Insights </strong> tab of your repository.</p>
<p>See: <a href="https://help.github.com/articles/viewing-and-updating-vulnerable-dependencies-in-your-repository/" rel="nofollow noopener">https://help.github.com/articles/viewing-and-updating-vulnerable-dependencies-in-your-repository/</a></p>

**Comment by lacostenycoder** [comment]
<p>I’m trying to find out how to reopen a dismissed security alert, but can’t find it on the link you provided. Any chance to provide instructions more clearlY?</p>

**Comment by mpboom** [comment]
<p>On a repository page you have several tabs, like  <strong>Code </strong> and  <strong>Pull requests </strong> for example. Head over to <strong>Insights </strong> to find the security alerts.</p>

**Comment by wrk4all** [comment]
<p>Hey did you find a way how to reinstate dismissed ones? </p>
<p>Thanks :) </p>

**Comment by mpboom** [comment]
<p><a class="mention" href="/wrk4all">@wrk4all</a> as you can see at the responses above, you are always able to view past security issues.</p>
<p>Could you elaborate on what you mean by “reinstating” dismissed ones?</p>

**Comment by wrk4all** [comment]
<p>The topic of this thread is about reopening/resintating a dismissed security alerts (Not notifications). So i asked if a way to reinstated the dismssied alerts has been found. (See below) </p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/e/e810450e9d15079d6d45399966bba279135b52d0.png" data-download-href="/uploads/short-url/x6VD7iJheQvm42oV2VNGDvGbt0Q.png?dl=1" title="Screen Shot 2019-08-14 at 11.45.12 AM.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/34996815/181070808-001c746f-d3c8-41a4-a146-8ee81bef4e28.png" alt="Screen Shot 2019-08-14 at 11.45.12 AM.png" data-base62-sha1="x6VD7iJheQvm42oV2VNGDvGbt0Q" width="690" height="172" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/e/e810450e9d15079d6d45399966bba279135b52d0_2_690x172.png, https://aws1.discourse-cdn.com/github/original/2X/e/e810450e9d15079d6d45399966bba279135b52d0.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/e/e810450e9d15079d6d45399966bba279135b52d0.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/e/e810450e9d15079d6d45399966bba279135b52d0_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2019-08-14 at 11.45.12 AM.png</span><span class="informations">794×199 15.3 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Thank you :) </p>

**Comment by mpboom** [comment]
<p>I see. Thanks for the clarification <a class="mention" href="/wrk4all">@wrk4all</a> ! There is no way that I know of, but maybe someone else does!</p>

**Comment by wrk4all** [comment]
<p>Boo hoo lol.</p>
<p>Mark you are the best bro, Thanks for being so responsive :o).</p>
<p>Best,</p>
<p>Benny.</p>

**Comment by EGrun** [comment]
<p>+1 need a solution to this issue.</p>

**Comment by chalbersma** [comment]
<p>Can this be marked not solved? This hasn’t. It’s still not possible to “undismiss” a security alert. You can view them but cannot change the dismissed status.</p>

**Comment by dinvlad** [comment]
<p>Are there any updates on this? Thanks</p>

**Comment by mehdiaws** [comment]
<p>The only “solution” for me was to remove the codes and add them again.</p>

**Comment by joxl** [comment]
<p>This is not a valid solution. I don’t know why this post got marked as solved. It’s clearly not.</p>

**Comment by mpboom** [comment]
<p>Hey <a class="mention" href="/joxl">@joxl</a> , this post was marked solved by the author as it was supposedly of good use to them. I agree that the comment provided by me doesn’t exactly cover the question most people may have when visiting this topic as I stated in my Aug '19 post.</p>
<p>As far as I’m aware, things are still the same. I would however suggest you to open up a new, fresh, topic if you’re still stuck on reopening these alerts.</p>

**Comment by recursivelycurious** [comment]
<p>I also bumped into this limitation.</p>
<p><em>Potential/partial work around.</em></p>
<p>I have been unable to find a way to re-activate or re-open a dismissed security alert, however, since you can navigate to the permalink for the advisory you can use that link or a screenshot of the content (since specific permissions are needed to access it) to open an appropriate item to track it elsewhere.</p>
<p><strong>Open new issues with care about what details are included.</strong></p>
<p>Since issues on the repo itself are visible to everyone who can access the repo (the world if it is public) if you have a more private way of tracking the issue to conclusion that may be best. If you must use GitHub issues, perhaps a sanitized / redacted statement of the solution without security aspects discussed could be opened if that is what is needed to mitigate the vuln.</p>
<p>For example, details of vulns aside, often dependabot finds items that are remediated with a package update, so an issue like the following might be sufficient.</p>
<pre><code class="lang-markdown"># Issue title
Update package foo to latest version

# Issue detail
We are using foo x.y and need a feature in foo x.z.
</code></pre>

**Comment by ianbtt** [comment]
<p>This is crazy. So anyone can just dismiss an alert? Surely the dismissal is audited somewhere - why else would you have to specify the reason for dismissal? If I can track and review the dismissals I can at least decide if I need to manually reinstate them?</p>
